tests/config/mod.rs

```diff
@@ -66,6 +66,9 @@ fn test_serde_custom_tikv_config() {
         end_point_max_tasks: 12,
         end_point_stack_size: ReadableSize::mb(12),
         end_point_recursion_limit: 100,
+        snap_min_write_bytes_per_time: ReadableSize::kb(64),
+        snap_max_write_bytes_per_time: ReadableSize::mb(1),
+        snap_max_write_bytes_per_sec: ReadableSize::mb(10),
```

Comment:
Enlarge default value of `snap_max_write_bytes_per_sec`.

Reply:
this is for the test, I think it is fine.